src/state/stake/hooks.ts

```diff
@@ -1745,7 +1735,7 @@ export function useStakingInfo(
             : valueOfTotalStakedAmountInBaseToken?.toSignificant();
 
           const perMonthReturnInRewards =
-            (Number(dQuickToQuick) * Number(quickPrice) * 30) /
+            (Number(stakingInfo.rate) * Number(dQuickPrice) * 30) /
```

Comment:
We need to fix the permonth rewards we should create a function to get the days in the current montj

Reply:
Updated 30 to `getDaysCurrentYear() / 12` to get correct average rewards per month